docs/src/man/getting_started.md

```diff
@@ -111,61 +123,202 @@ julia> nulls(Int, 1, 3)
 
 The `DataFrame` type can be used to represent data tables, each column of which is a vector. You can specify the columns using keyword arguments or pairs:
 
-```julia
-df = DataFrame(A = 1:4, B = ["M", "F", "F", "M"])
-df = DataFrame(:A => 1:4, :B => ["M", "F", "F", "M"])
+```jldocttest
+julia> using DataFrames
+
+julia> df = DataFrame(A = 1:4, B = ["M", "F", "F", "M"])
+4×2 DataFrames.DataFrame
+│ Row │ A │ B │
+├─────┼───┼───┤
+│ 1   │ 1 │ M │
+│ 2   │ 2 │ F │
+│ 3   │ 3 │ F │
+│ 4   │ 4 │ M │
+
+julia> df == DataFrame(:A => 1:4, :B => ["M", "F", "F", "M"])
+true
+
 ```
 
 It is also possible to construct a `DataFrame` in stages:
 
-```julia
-df = DataFrame()
-df[:A] = 1:8
-df[:B] = ["M", "F", "F", "M", "F", "M", "M", "F"]
-df
+```jldocttest
+julia> using DataFrames
+
+julia> df = DataFrame()
+0×0 DataFrames.DataFrame
+
+
+julia> df[:A] = 1:8
+1:8
+
+julia> df[:B] = ["M", "F", "F", "M", "F", "M", "M", "F"]
+8-element Array{String,1}:
+ "M"
+ "F"
+ "F"
+ "M"
+ "F"
+ "M"
+ "M"
+ "F"
+
+julia> df
+8×2 DataFrames.DataFrame
+│ Row │ A │ B │
+├─────┼───┼───┤
+│ 1   │ 1 │ M │
+│ 2   │ 2 │ F │
+│ 3   │ 3 │ F │
+│ 4   │ 4 │ M │
+│ 5   │ 5 │ F │
+│ 6   │ 6 │ M │
+│ 7   │ 7 │ M │
+│ 8   │ 8 │ F │
+
 ```
 
-The `DataFrame` we build in this way has 8 rows and 2 columns. You can check this using `size` function:
+The `DataFrame` we build in this way has 8 rows and 2 columns. You can check this using the
+`size`, `nrow`, and `ncol` functions:
```

Comment:
Better not mention `nrow` and `ncol` as there have been lots of discussions about removing these.